inyoka_theme_default/templates/portal/configuration.html

```diff
@@ -0,0 +1,41 @@
+{#
+    templates.portal.configuration
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    On this page the administrator can set general configuration values.
+
+    :copyright: (c) 2013-2017 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+{% extends 'portal/base.html' %}
+
+{% block title %}
+  {% trans %}Configuration{% endtrans %} – {{ super() }}
+{% endblock title %}
+
+{% block breadcrumb %}
+  {{ super() }}
+
+  {{ macros.breadcrumb_item(_('Configuration'), href('portal', 'config')) }}
+{% endblock %}
+
+{% block content %}
+  {% call macros.outer_form(csrf_token(), form, manually_rendered=True) %}
+    <h1>{% trans %}General settings{% endtrans %}</h1>
+    {{ macros.inner_form(form, ['global_message', 'welcome_message',
+        'blocked_hosts', 'wiki_newpage_template', 'wiki_newpage_root',
+        'wiki_newpage_infopage', 'wiki_edit_note', 'license_note',
+        'ikhaya_description', 'planet_description', 'team_icon']) }}
```

Comment:
What you think about grouping the wiki settings ( 'wiki_newpage_template', 'wiki_newpage_root',  'wiki_newpage_infopage', 'wiki_edit_note' ) and the ikhaya ones ( 'ikhaya_description', 'planet_description') into there own sub sections like it is done for the release count down?